src/nvim/ui.h

```diff
@@ -22,6 +23,7 @@ struct ui_t {
   void (*clear)(UI *ui);
   void (*eol_clear)(UI *ui);
   void (*cursor_goto)(UI *ui, int row, int col);
+  void (*cursor_style_set)(UI *ui, Dictionary args); // or a Dictionary ?
```

Comment:
I forgot to ask this one: I don't think Dictionary will be available to other UIs so what should I use ?
(Dictionary defined in https://github.com/neovim/neovim/blob/9ef4be9aab6a9a16cf185ca17a2dabe87f8aa328/src/nvim/api/private/defs.h)